packages/ocaml-variants/ocaml-variants.5.0.0+options/opam

```diff
@@ -85,7 +85,8 @@ build: [
     "--enable-frame-pointers" {ocaml-option-fp:installed}
     "CC=cc" {!ocaml-option-32bit:installed & !ocaml-option-musl:installed & (os="openbsd"|os="macos")}
     "CC=musl-gcc" {ocaml-option-musl:installed & os-distribution!="alpine"}
-    "CFLAGS=-Os" {ocaml-option-musl:installed}
+    "CFLAGS=-Os" {ocaml-option-musl:installed & arch!="arm64"}
+    "CFLAGS=-Os -mno-outline-atomics" {ocaml-option-musl:installed & arch="arm64"}
```

Comment:
AFAICT this is going to break on Debian 10 because it ships a version of GCC that's too old.

Is this a non-issue for some reason? E.g., if `ocaml-option-musl:installed` is impossible on debian 10.

I don't think it's a hard blocker… and if it's a blocker then the proper fix would be to introduce some constraint on gcc versions which we maybe don't want to do anyway?

Reply:
Debian 10 is outside of LTS so let's say it's not a blocker. If people start having issues on debian 10 we'll see what to do.